mods/ts/maps/sunstroke/rules.yaml

```diff
@@ -0,0 +1,36 @@
+World:
+	GlobalLightingPaletteEffect:
+		Red: 1.6
+		Blue: 0.9
+		Green: 1.2
+		Ambient: 0.3
+	LuaScript:
+		Scripts: sunstroke.lua
+
+GAOLDCC2:
+	Capturable:
+	ProvidesPrerequisite@BuildingName:
+
+GAOLDCC3:
```

Comment:
They definitely should be remapable (they were in TS). If we want them to have a specific initial color on some maps, we can simply set up a custom 'CreepsBase' player with the desired player color.